rm -f _test/github.com/jacobsa/ogletest/foobar.a
6g -I/Users/jacobsa/clients/ogletest/_obj  -p github.com/jacobsa/ogletest/foobar -o _gotest_.6 panicking_test.go 
rm -f _test/github.com/jacobsa/ogletest/foobar.a
gopack grc _test/github.com/jacobsa/ogletest/foobar.a _gotest_.6 
[----------] Running tests from PanickingTest
[ RUN      ] PanickingTest.PanickingTest
panicking_test.go:39:
panic: foobar

/Users/jacobsa/clients/ogletest/run_tests.go:68 (0x26335)
	com/jacobsa/ogletest._func_001: "panic: %v\n\n%s", r, debug.Stack())
/Users/jacobsa/go/src/pkg/runtime/proc.c:1386 (0x12404)
	panic: reflect·call(d->fn, d->args, d->siz);
/private/var/folders/00/0mb18000h01000cxqpysvccm002jc5/T/ogletest_integration_test222285714/panicking_test.go:39 (0x18727)
	(*PanickingTest).PanickingTest: panic("foobar")
/Users/jacobsa/go/src/pkg/reflect/value.go:517 (0x61e13)
	Value.call: call(fn, unsafe.Pointer(ptr), uint32(size))
/Users/jacobsa/go/src/pkg/reflect/value.go:330 (0x60c22)
	Value.Call: return v.call("Call", in)
/Users/jacobsa/clients/ogletest/run_tests.go:188 (0x25eb1)
	com/jacobsa/ogletest.runMethodIfExists: method.Call([]reflect.Value{})
/Users/jacobsa/clients/ogletest/run_tests.go:80 (0x25313)
	com/jacobsa/ogletest.runTest: runMethodIfExists(suiteInstance, method.Name)
/Users/jacobsa/clients/ogletest/run_tests.go:146 (0x2595c)
	com/jacobsa/ogletest.runTestsInternal: failures := runTest(suite, method)
/Users/jacobsa/clients/ogletest/run_tests.go:105 (0x2646e)
	com/jacobsa/ogletest._func_002: runTestsOnce.Do(func() { runTestsInternal(t) })
/Users/jacobsa/go/src/pkg/sync/once.go:40 (0x5bdcd)
	(*Once).Do: f()
/Users/jacobsa/clients/ogletest/run_tests.go:105 (0x25394)
	com/jacobsa/ogletest.RunTests: runTestsOnce.Do(func() { runTestsInternal(t) })
/private/var/folders/00/0mb18000h01000cxqpysvccm002jc5/T/ogletest_integration_test222285714/panicking_test.go:32 (0x186db)
	TestOgletest: func TestOgletest(t *testing.T) { RunTests(t) }
/Users/jacobsa/go/src/pkg/testing/testing.go:189 (0x1aa95)
	tRunner: test.F(t)
/Users/jacobsa/go/src/pkg/runtime/proc.c:258 (0x105a6)
	goexit: runtime·goexit(void)


[  FAILED  ] PanickingTest.PanickingTest
[ RUN      ] PanickingTest.ZzzSomeOtherTest
[       OK ] PanickingTest.ZzzSomeOtherTest
[----------] Finished with tests from PanickingTest
--- FAIL: oglematchers_test.TestOgletest (0.06 seconds)
FAIL
gotest: "./6.out" failed: exit status 1
